packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -203,7 +203,7 @@ const Dashboard = ({
                 setOnBoardTourRunState(true);
             } else {
                 setHasTourEnded(false);
-                if (show_tour_dialog_mobile || show_tour_dialog_desktop) {
+                if ((show_tour_dialog_mobile || show_tour_dialog_desktop) && !is_strategy_modal_open) {
```

Comment:
something like this 
if ((show_tour_dialog_mobile || show_tour_dialog_desktop){ 
if(!is_strategy_modal_open) ) {
 setTourDialogVisibility(true);
}
}

Reply:
it works the same, my syntax is shorter only